src/RPackage-Core/RPackage.class.st

```diff
@@ -1522,6 +1522,17 @@ RPackage >> tagsForClasses [
 	^self classTags reject: [:each | each isRoot] thenCollect: [:each | each name]
 ]
 
+{ #category : #testing }
+RPackage >> testSuite [
```

Comment:
I think this should be repackaged closer to the tests or the user, right?

Reply:
Yes, I will change